internal/command/generate.go

```diff
@@ -47,19 +49,31 @@ var CmdGenerate = &Command{
 		if err := validateRequiredFlag("api-root", flagAPIRoot); err != nil {
 			return err
 		}
-
 		apiRoot, err := filepath.Abs(flagAPIRoot)
 		if err != nil {
 			return err
 		}
-
 		outputDir := filepath.Join(ctx.workRoot, "output")
 		if err := os.Mkdir(outputDir, 0755); err != nil {
 			return err
 		}
 		slog.Info(fmt.Sprintf("Code will be generated in %s", outputDir))
 
-		if err := container.GenerateRaw(ctx.containerConfig, apiRoot, outputDir, flagAPIPath); err != nil {
+		// check if the library exists in the remote pipeline state, if so use GenerateLibrary command
+		// otherwise use GenerateRaw command
+		// In case of error when looking up library, we will fallback to GenerateRaw command
+		// and log the error
+		libraryId := checkIfLibraryExistsInLanguageRepo(ctx)
+		if libraryId != "" {
+			ctx.languageRepo, err = cloneOrOpenLanguageRepo(ctx.workRoot)
+			slog.Warn("Unable to checkout language repo ", "error", err)
```

Comment:
I think you're missing a condition here :) (And returning an error if we can't check out the repo.)